loops.js

```diff
@@ -32,7 +38,15 @@
 
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over..
 // ==========================================
-
+for (let numberX = 0; numberX < 16; numberX++) {
+    if (numberX === 0) {
+        console.log(numberX + " is even");
+    } else if (numberX % 2 === 0) {
+        console.log(numberX + " is even");
+    } else {
+        console.log(numberX + " is oneven");
+    }
+}
```

Comment:
De eerste if is overbodig omdat 0 % 2 ook nul als uitkomst geeft, maar kan geen kwaad want op deze wijze werkt hij ook prima.